vscode-dotnet-runtime-library/src/Utils/WebRequestWorker.ts

```diff
@@ -171,12 +204,19 @@ export class WebRequestWorker {
                 }
                 else
                 {
-                    formattedError = new Error(`Please ensure that you are online: Request to ${this.url} Failed: ${formattedError.message}`);
+                    formattedError = new Error(`Please ensure that you are online: Request to ${this.url} Failed: ${formattedError.message}.
+If you are on a corporate proxy with a result of 400, you may need to manually set the proxy in our extension settings: Please see: https://github.com/dotnet/vscode-dotnet-runtime/blob/main/Documentation/troubleshooting-runtime.md
```

Comment:
```suggestion
If you are on a corporate proxy with a result of 400, you may need to manually set the proxy in our extension settings. Please see https://github.com/dotnet/vscode-dotnet-runtime/blob/main/Documentation/troubleshooting-runtime.md for more details.
```

Minor grammar suggestion here.